src/Controllers/NIPSWeb/confirm_central_upload.php

```diff
@@ -11,7 +11,7 @@
     $_REQUEST['artist'],
     $_REQUEST['album'],
     $_REQUEST['position'],
-    $_REQUEST['explicit'] ? true : null
+    filter_var($_REQUEST['explicit'], FILTER_VALIDATE_BOOLEAN) ? true : null
```

Comment:
What was there directly before it did not work, because "false" the string evaluates to true.